authorization-for-rag/langchain-next-js/src/lib/tools/context-docs.ts

```diff
@@ -6,7 +6,7 @@ import { getVectorStore } from '@/lib/rag/embedding';
 
 export const getContextDocumentsTool = tool(
   async ({ question }, config) => {
-    const user = config?.configurable?._credentials?.user;
+    const user = config.configurable?.langgraph_auth_user;
```

Comment:
could also remain as `const user = config?.configurable?._credentials?.user;`, since we are sending both now after passthrough updates in:
https://github.com/auth0-samples/auth0-ai-samples/pull/11/files#diff-c9bdef2bfde99090ff29d0ff4326570e827a75a3de70ad3e4a6cd0c3b879f255

passthrough is:
https://github.com/auth0-samples/auth0-ai-samples/blob/main/authorization-for-rag/langchain-next-js/src/app/api/chat/%5B..._path%5D/route.ts#L5-L30